borg/helpers.py

```diff
@@ -658,7 +662,7 @@ def format_archive(archive):
     return '%-36s %s [%s]' % (
         archive.name,
         format_time(to_localtime(archive.ts)),
-        hexlify(archive.id).decode('ascii'),
+        bin_to_hex(archive.id),
```

Comment:
i first had that and it failed - because "archive" is in reality an ArchiveInfo instance.